ts/create-kpt-functions/package.json

```diff
@@ -41,25 +41,25 @@
     "go-npm": "^0.1.9",
     "is-valid-npm-name": "0.0.4",
     "mustache": "^3.0.1",
-    "request": "^2.88.0",
+    "request": "^2.88.2",
```

Comment:
why is this file changes as part of this PR?

Reply:
Removed